### ISSUE TITLE
Document on why it is fast and efficient

### ISSUE THREAD

**Issue by myselfhimself** [mrachinskiy/booltron#5]
Hello,
Congratulation for your fresh work and introductory videos.
Blender's current Boolean modifier creates sometimes non-closed/manifold meshes, which your add-on tells it checks for.
To fix that, I tend to use the Cork-on-blender add-on, which uses a different back-end as Blender's own boolean solver. (See the most up to date fork here: https://github.com/neworigin2016/cork-on-blender)
It would be great to explain a bit more technically in your README file, how your add-on manages to fix the issues you put forward.
For example, while your add-on keeps using Blender's native boolean modifier in background (I have read through your code very quickly), how do you manage to have fully manifold results?
Basically your plugin allows multiple rather just two boolean operands to be used, to me this is an extra feature, which could be written down.

**Comment by mrachinskiy** [comment]
Hi, Booltron does not fix the result of a boolean operation, it's just checks for non-manifolds and pops a warning if it finds any.
